app/main.py

```diff
@@ -1,3 +1,71 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> int:
```

Comment:
Please fix type annotations for func arguments everwhere, try smtn like this Distance | int | float etc.